lib/ui/window/window.cc

```diff
@@ -84,7 +84,8 @@ void ReportUnhandledException(Dart_NativeArguments args) {
 Dart_Handle SendPlatformMessage(Dart_Handle window,
                                 const std::string& name,
                                 Dart_Handle callback,
-                                const tonic::DartByteData& data) {
+                                Dart_Handle data_handle) {
+  tonic::DartByteData data(data_handle);
```

Comment:
The scope of the `tonic::DartByteData` can be restricted further:

* Move the `tonic::DartByteData` construction into the block that calls `HandlePlatformMessage`
* Remove the `data.Release()` call earlier in the function
* Pass `data_handle` to `Dart_IsNull`

Reply:
Good call. Done.